addons/mail/security/ir.model.access.csv

```diff
@@ -46,6 +46,14 @@ access_mail_shortcode,mail.shortcode,model_mail_shortcode,base.group_user,1,1,1,
 access_mail_shortcode_portal,mail.shortcode.portal,model_mail_shortcode,base.group_portal,1,0,0,0
 access_mail_activity_all,mail.activity.all,model_mail_activity,,0,0,0,0
 access_mail_activity_user,mail.activity.user,model_mail_activity,base.group_user,1,1,1,1
+access_mail_activity_plan_all,mail.activity.plan.all,model_mail_activity_plan,,0,0,0,0

```

Comment:
Hello, just passing by, note that `0,0,0,0` rules have no effect, also rules without groups are deprecated generally speaking (https://github.com/odoo/odoo/pull/118701).